src/compas/scene/scene.py

```diff
@@ -6,58 +6,95 @@
 from .context import clear
 
 
+class SceneObjectNode(TreeNode):
+    def __init__(self, sceneobject):
+        super(SceneObjectNode, self).__init__(name=sceneobject.name)
+        self.object = sceneobject
+
+    @property
+    def parent_object(self):
+        if self.parent and isinstance(self.parent, SceneObjectNode):
+            return self.parent.object
+        return None
+
+    @property
+    def children_objects(self):
```

Comment:
child_objects

Reply:
or just `def children()` and `def parent()`?

Reply:
those names already exist and refer to the parnet/child nodes. this is about the objects contained in the nodes